src/redux/actions/assetRegisterApi.js

```diff
@@ -135,6 +136,17 @@ export const getMoreAssets = (url) => ({
   }
 });
 
+/**
+ * Reset the asset list to the initial "unfetched" state.
+ *
+ * The current query state is not changed. The effect of this action is performed whenever an asset
+ * is modified or created via putAsset or postAsset so there's no need to fire this action if
+ * modifying assets.
+ */
+export const resetAssets = () => ({
```

Comment:
do we need this?

Reply:
ATM, no, but I feel that an explicit way to do this is an obvious hole if it weren't here.